python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -656,13 +656,21 @@ def radius_servers(self) -> None:
         if not self.inputs.aaa_settings.radius:
             return
 
+        seen_server_combinations: set[tuple[str, str]] = set()
         for server in self.inputs.aaa_settings.radius.servers:
             server_vrf, source_interface = self._get_vrf_and_source_interface(
                 vrf_input=server.vrf,
                 vrfs=self.inputs.aaa_settings.radius.vrfs,
                 set_source_interfaces=True,
                 context=f"aaa_settings.radius.servers[host={server.host}].vrf",
             )
+            # TODO: Temporary workaround to avoid duplicate configs; replace once schema supports unique (host, VRF) combinations.
+            server_key: tuple[str, str] = (server.host, server_vrf)
+            if server_key in seen_server_combinations:
```

Comment:
No need of `server_key` we can check directly
```suggestion
            if (server.host, server_vrf) in seen_server_combinations:
```

Reply:
updated the code, now we need server_key  as used multiple time.